DeploymentCloud/Deployment.JobRunner/deployResources.ps1

```diff
@@ -0,0 +1,89 @@
+param(
+    [string]
+    $tenantId,
+    [string]
+    $subscriptionId,
+    [string]
+    $resourceGroupName,
+    [string]
+    $applicationId,
+    [string]
+    $appSecretKey,
```

Comment:
how do we get the values of these parameters

Reply:
Those values are retrieved from previous data-accelerator deployment. More specifically, those are the chosen values from common.parameters.txt and admin.parameters.txt values, and then ones generated for the deployment script such the keyvault prefix. At this moment, the user should look for those values at the target environment where data-accelerator is deployed. I added those details at the README.MD file